src/App.jsx

```diff
@@ -14,45 +16,81 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_FIELD_ALPHABET = 'alphabet';
+const SORT_FIELD_LENGTH = 'length';
+
+export const App = () => {
+  const [sortField, setSortField] = useState('');
+  const [reverseField, setReverseField] = useState(false);
+  const goods = [...goodsFromServer];
+
+  const onClickReverse = () => {
+    setReverseField(!reverseField);
+  };
+
+  if (sortField === SORT_FIELD_ALPHABET) {
+    goods.sort();
+  }
+
+  if (sortField === SORT_FIELD_LENGTH) {
+    goods.sort((firstGood, nextGood) => firstGood.length - nextGood.length);
+  }
+
+  if (reverseField) {
+    goods.reverse();
+  }
```

Comment:
it's better to create separate function with arguments and return prepared goods. because now you have `const` variable and mutate it which is not good approach